### ISSUE TITLE
Categories dropdown has no focus state

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-news-2021#103]
When keyboard-navigating through the header, after the nav menu, the focus disappears. If I hit space or enter the categories list toggles, so I assume I'm on the hidden checkbox here.

<img width="270" alt="Screen Shot 2021-12-02 at 4 04 44 PM" src="https://user-images.githubusercontent.com/541093/144502982-591263c6-8ac2-4366-95d0-cad655b51edd.png">

This should get a focus state so it's clear, it can be done with a similar sibling selector:

```css
.local-header__categories-toggle:focus  ~ label {
    outline: 1px dotted currentColor;
}
```

**Comment by tellyworth** [comment]
I made a PR for this. It needs some styling (see screenshot) and probably further work, but finishing it can wait till after SotW.
<img width="210" alt="Screen Shot 2021-12-06 at 5 17 52 pm" src="https://user-images.githubusercontent.com/7200686/144797104-100173b4-4b6c-4e5e-b16b-fe3ee697ace0.png">

**Comment by ryelle** [comment]
Fixed in #106
